src/app/pages/questions/components/finish/finish.component.ts

```diff
@@ -68,4 +71,23 @@ export class FinishComponent implements OnChanges {
   toggleChanged(event) {
     this.completedInClinic = event
   }
+
+  getEtaText(progress) {
+    const duration = getMinutes({ milliseconds: Date.now() - this.startTime })
+    return (
+      'About ' + (duration * (100 - progress)) / progress + ' minutes remaining'
```

Comment:
Similar to `radar-output`.

Reply:
isn't the progress count in the range [0,1] but here seems like the calculation needs it to be a percentage [0,100]?

Reply:
Yup updated to `progressDisplay` variable.